bp/src/main.m

```diff
@@ -8,68 +8,29 @@
 //  WITHOUT WARRANTIES OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
 
 #import <Foundation/Foundation.h>
-#import "CoreSimulator.h"
-#import "BPStats.h"
-#import "BPWriter.h"
-#import "BPConfiguration.h"
 #import "Bluepill.h"
+#import "BPConfiguration.h"
+#import "BPSimulator.h"
 #import "BPUtils.h"
 #import "SimulatorHelper.h"
+#import "BPStats.h"
+#import "BPWriter.h"
 
 #import <getopt.h>
 #import <libgen.h>
 
 int main(int argc, char * argv[]) {
     @autoreleasepool {
-        // This next part is because we're weak-linking the private Xcode frameworks.
```

Comment:
don't we need this any more ?

Reply:
We do, it's in an API (`[BPUtils setupWeakLinking:]`) because when `bluepill` was made to use internal APIs, it didn't add this so it was unclear which frameworks it was finding. Putting it behind an API allowed me to reuse the same code for both `bluepill` and `bp`.